### PR TITLE
Fixing issue #2035

### PR DESCRIPTION
**Pull request by mani1996**
Changed log type to warning instead of error

### CODE REVIEW

**Comment by mani1996** [comment]
What is meant by "0.00% of diff hit" ? Is there something I need to change in my code?

**Comment by redapple** [comment]
@mani1996 , it means that there's no test covering your change.
If you want, you can try adding a test case for this, but I doubt it's crucial

**Comment by kmike** [comment]
The PR looks good to me, thanks @mani1996! But it seems something is changed in scrapy master, so it can't be merged as-is - could you please rebase it to make mergeable?

File: scrapy/core/scheduler.py [hunk 1]
```diff
@@ -86,7 +86,7 @@ def _dqpush(self, request):
             self.dqs.push(reqd, -request.priority)
         except ValueError as e: # non serializable request
             if self.logunser:
-                logger.error("Unable to serialize request: %(request)s - reason: %(reason)s",
+                logger.warning("Unable to serialize request: %(request)s - reason: %(reason)s",
```
Comment (Digenis) [inline scrapy/core/scheduler.py]: indent the next 2 lines by 2 spaces each

**Comment by redapple** [comment]
This change has been applied on top of master in https://github.com/scrapy/scrapy/pull/2269
Thanks again @mani1996

**Pull request closed without merging**